.github/actions/build-engine/action.yaml

```diff
@@ -0,0 +1,109 @@
+# .github/actions/build-engine/action.yml
+name: ""
+description: "Build the engine with a specific gclient setup and config name"
+inputs:
+
+  gclient:
+    description: "name of gclient configuration to use"
```

Comment:
Let's update the description with some breadcrumbs. What is likely the correct choice? Where might someone find the options?